js/main.js

```diff
@@ -19,7 +24,76 @@ function saveEntry(event) {
   data.entries.unshift(newEntry);
   $img.setAttribute('src', '../images/placeholder-image-square.jpg');
   $form.reset();
+
+  var addEntry = createEntryTree(newEntry);
+  $entryList.prepend(addEntry);
+  $noEntry.className = '.no-entry hidden';
```

Comment:
I don't think you meant to have a `.` in the value here.

Reply:
Oops, must've gotten my mind switched up a little when making the DOM elements. Made the change!

![image](https://user-images.githubusercontent.com/109567499/190014148-49172e64-cdf0-487b-9e4a-0ba4635a749d.png)